deploy/helm/sumologic/values.yaml

```diff
@@ -52,7 +52,7 @@ sumologic:
     # config_context_auth_info:
     # config_context_cluster:
     token: "${file(\"/var/run/secrets/kubernetes.io/serviceaccount/token\")}"
-    # load_config_file:
+    load_config_file: false
```

Comment:
fix for https://github.com/terraform-providers/terraform-provider-kubernetes/issues/821

Reply:
i am also facing same issue as it is being running via github action workflow